kythe/cxx/indexer/cxx/testdata/df/df_var_influence.cc

```diff
@@ -0,0 +1,17 @@
+// Basic var to var influence.
+void f() {
+  //- @x defines/binding VarX
+  //- @y defines/binding VarY
+  //- @z defines/binding VarZ
+  int x = 0, y = 1, z = 2;
+  //- VarZ influences VarY
+  //- VarY influences VarX
+  //- !{VarZ influences VarX}
+  //- !{VarY influences VarZ}
+  x = y = z;
```

Comment:
I assume we're punting on references for now? e.g.
```
int& a = b;
a = c; // c influences a && b
```

Reply:
Yes.